tests/js/spec/views/eventsV2/table/arithmeticInput.spec.tsx

```diff
@@ -180,4 +180,13 @@ describe('ArithmeticInput', function () {
     input.simulate('blur');
     expect(query).toEqual(`transaction.duration + measurements.lcp `);
   });
+
+  it('handles autocomplete on invalid term', function () {
+    const input = wrapper.find('input');
+    input.simulate('focus');
+
+    const value = 'foo + bar';
+    input.simulate('change', {target: {value}});
+    input.simulate('keydown', {key: 'ArrowDown'});
+  });
```

Comment:
This test seems to be missing an assertion.

Reply:
Not sure what to assert here, I'm only trying to validate that this sequence of actions doesn't raise an error

Reply:
You can make an assertion of what the UI should expect to look like for this edge case.

Reply:
@dashed added one in eff7932